application/Handler.php

```diff
@@ -112,7 +112,7 @@ public function authorize()
         } else {
             if ($media->is('restricted')) {
                 if (\AAM::api()->getConfig(
-                    'addon.protected-media-files.settings.deniedRedirect', false
```

Comment:
Hello @rob1998,

Actually there is the dedicated setting on the AAM Settings page that enabled the redirect. It is called `Invoke Access Denied Redirect`. Have you tried to enable it?

<img width="1440" alt="Screen Shot 2023-01-20 at 8 30 26 AM" src="https://user-images.githubusercontent.com/34006503/213706577-b379fa43-5895-4ad2-9dd0-5f757881649f.png">